### DISCUSSION TITLE
Update >> spring-integration-sftp >> jsch >> 0.1.55 to 0.1.72

### DISCUSSION THREAD

**Discussion by dmerinom** [spring-projects/spring-integration#3824] [Q&A]
Dear Partner

We have noticed that in the spring-integration-sftp package it makes use of the library:

<dependency>
     <groupId>com.jcraft</groupId>
     <artifactId>jsch</artifactId>
     <version>0.1.55</version>
</dependency>

Which is not updated since Nov, 2018
But this library has problems to connect with Azure and to solve it people are using this library:

<dependency>
     <groupId>com.github.mwiede</groupId>
     <artifactId>jsch</artifactId>
     <version>0.1.72</version>
</dependency>

But it is not the official one and we do not trust to introduce it in our code

Do you have a solution in mind to clear up this problem soon?

Best Regards

**Comment by artembilan** [comment]
See this one: https://github.com/spring-projects/spring-integration/issues/3572.

Not sure what to ad else, unless we indeed going to upgrade the currently in progress `6.0` to that third-party fork.
